en/sql/tuning.rst

```diff
@@ -3829,6 +3829,89 @@ The join with the *right_tbl* table was eliminated.
                 9           60          600  'Left- 806'                     6
                10           60          600  'Left- 906'                     6
 
+.. _pred-push:
+
+Predicate Push
+-----------------------
+
+**Predicate Push** is an optimization that pushes the predicate of the main query into the view.
+
+By applying **Predicate Push** to the subquery, the amount of data queried in the main query is reduced.
```

Comment:
The meaning of this sentence is not wrong if we consider the main query as the entire query.
But I think it reduces the amount of data being queried in the view query out of the entire query.
So, I would like to suggest that it would be better to mention the view query rather than the main query.